internal/pkg/runtime/engine/singularity/container_linux.go

```diff
@@ -1267,6 +1286,7 @@ func (c *container) addBindsMount(system *mount.System) error {
 		if err != nil {
 			return fmt.Errorf("unable to add %s to mount list: %s", src, err)
 		}
+		system.Points.AddRemount(mount.BindsTag, dst, flags)
```

Comment:
Is the addition here fixing a separate issue?

Also, in the contain case above you are returning any error from `AddRemount`, but here it is not caught / returned. Does it need to be?

Reply:
Yes this is a different issue but while we are in this code part it's worth to fix it. Fixed the return.